notebooks/failure-type-classification/stage/fail_type_functions.ipynb

```diff
@@ -0,0 +1,343 @@
+{
```

Comment:
<p>Can we move all of these out of <code>stage</code>?  and update all the paths accordingly? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/284/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>